packages/time-picker/test/events.test.js

```diff
@@ -81,70 +31,6 @@ describe('events', () => {
       timePicker.value = '02:00';
       expect(changeSpy.calledOnce).to.be.true;
     });
-
-    it('should not be fired on Enter if the value has not changed', () => {
-      timePicker.value = '01:00';
-      timePicker.open();
-      enter(inputElement);
-      expect(changeSpy.called).to.be.false;
-    });
-
-    it('should not be fired on revert', () => {
-      timePicker.open();
-      timePicker.value = '01:00';
-      esc(inputElement);
-      esc(inputElement);
-      expect(changeSpy.called).to.be.false;
-    });
-
-    it('should be fired only once', async () => {
-      timePicker.focus();
-      timePicker.open();
-      await sendKeys({ type: '0' });
-      enter(inputElement);
-      inputElement.blur();
-      expect(changeSpy.callCount).to.equal(1);
-    });
-
-    it('should not be fired again on Enter if the value has not changed', async () => {
```

Comment:
This test was introduced in https://github.com/vaadin/web-components/pull/6396. However, I decided to drop it because the new tests already cover the changes from that PR: if I revert the changes, the new tests start to fail.